provider/image.go

```diff
@@ -253,7 +254,10 @@ func (p *dockerNativeProvider) dockerBuild(ctx context.Context,
 
 	//Start a session for BuildKit
 	if build.BuilderVersion == defaultBuilder {
-		sess, _ := session.NewSession(ctx, "pulumi-docker", "")
+		sess, err := session.NewSession(ctx, "pulumi-docker", identity.NewID())
```

Comment:
I tried to check https://github.com/pulumi/pulumi-docker/pull/450 to understand whether the error was ignored on purpose, but I couldn't. @guineveresaenger do you remember?

Reply:
It does not seem like we ignored this error on purpose.  Explicitly returning this error looks correct to me. 👍